compressor/tests/test_filters.py

```diff
@@ -234,12 +234,6 @@ def test_css_no_hash(self):
         filter = CssAbsoluteFilter(content)
         self.assertEqual(output, filter.input(filename=filename, basename='css/url/test.css'))
 
-        with self.settings(COMPRESS_URL='http://static.example.com/'):
```

Comment:
I would not consider that duplication. Testing with `COMPRESS_URL` being an absolute URL has merit. Same for similar instances below.